python/tk_desktop/desktop_engine_project_implementation.py

```diff
@@ -131,6 +134,13 @@ def _register_commands(self):
                 time.sleep(0.5)
 
             self._project_comm.call("trigger_register_command", name, gui_properties, groups)
+            project_command = {"name": name}
+            project_command.update(gui_properties)
+            project_commands.append(project_command)
+
+        # Let the proxy know command registration is complete and which
+        # commands were registered for this project.
+        self._project_comm.call_no_response("project_commands_finished", project_commands)
```

Comment:
Passing the list of commands that were registered because I couldn't figure out a reliable way to determine what was added from the desktop_window widget.  If there is in fact a way to figure out that information from the desktop_widget, then this isn't necessary.